packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -103,10 +103,14 @@ class FrameFlameChart extends CoreElement {
 
       row = maxRow;
 
-      row += 2;
+      row++;
     }
 
     drawCpuEvents();
+
+    // Add an additional row for spacing between CPU and GPU events.
```

Comment:
nit: instead of adding a row, add a specific number of pixels of padding.

Reply:
Added a todo because the row value is currently used in the drawing logic. I'd eventually like to put the CPU and GPU events in their own container, so I'll fix this at that point.